cranelift/codegen/meta/src/gen_inst.rs

```diff
@@ -159,7 +159,7 @@ fn gen_arguments_method(formats: &[Rc<InstructionFormat>], fmt: &mut Formatter,
 /// - `pub fn eq(&self, &other: Self, &pool) -> bool`
 /// - `pub fn hash<H: Hasher>(&self, state: &mut H, &pool)`
 fn gen_instruction_data_impl(formats: &[Rc<InstructionFormat>], fmt: &mut Formatter) {
-    fmt.add_block("impl InstructionData",|fmt| {
+    fmt.add_block("impl InstructionData", |fmt| {
```

Comment:
It's weird that this missing space didn't trigger a formatting error during CI prior to this; is it possible this crate isn't being checked?

Reply:
I think the whole crate disabled `rustfmt` or something, but I haven't looked into it yet.